std/experimental/allocator/building_blocks/fallback_allocator.d

```diff
@@ -247,7 +249,10 @@ struct FallbackAllocator(Primary, Fallback)
 
     Returns: $(D primary.empty & fallback.empty)
     */
-    static if (hasMember!(Primary, "empty") && hasMember!(Fallback, "empty"))
+    static if (hasMember!(Primary, "empty")
+               && is(typeof(Ternary(primary.empty)))
```

Comment:
Why this? We assume that if someone defines "empty" it must return `bool`. True, that assumption is not verified but we will insert all verifications later in a single place. So no need to attach tests to each constraint.